test/loopback-dl.test.js

```diff
@@ -1473,6 +1483,20 @@ describe('DataAccessObject', function() {
     assert(error, 'An error should have been thrown');
   });
 
+  it('should throw when malformed logical operators follow valid logical clauses', function() {
```

Comment:
@doublemarked I would say don't worry about it since you're are following the same consistency within a file.

@Amir-61 I noticed this issue coming a lot in PRs. We should just accept the change if they are consistent within the file and send follow up PRs to remove all the "shoulds" before enforcing it on the community. @bajtos Thoughts?